spec/requests/govspeak_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe GovspeakController do
+  subject(:rendered_page) { create_user && make_request && response }
+
+  let(:create_user) { create :user, permissions: ['signin', 'HMRC Editor'] }
+
+  describe 'POST #govspeak' do
+    let :make_request do
```

Comment:
this test is smart, but I am wondering if it might be easier to understand with a standard `before do ... end`,
instead of using let to make the request

Reply:
moved to using before blocks - I've got into that pattern because it makes the 'chain of tasks' lazy and run as part of the `is_expected.to` - this is useful in nested contexts but doesn't add anything here so changed to using `before`s